src/CFamily/CFamily.csproj

```diff
@@ -29,6 +29,11 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>Resources.resx</DependentUpon>
     </Compile>
+    <Compile Update="Subprocess\CFamilyStrings.Designer.cs">
```

Comment:
To reduce the number of file edits, I created a new resources file under the new "Subprocess" folder with the same name as the existing Integration.Vsix resources file. I can rename it later, along with the namespaces.

The new file contains just the subprocess runner-specific strings.